Code/Framework/AzCore/AzCore/Component/Entity.cpp

```diff
@@ -600,7 +600,7 @@ namespace AZ
     }
 
     Component* Entity::FindComponent(ComponentId id) const
-    {
+                      {
```

Comment:
nit: That's a lot of whitespace 😅 Likely worth doing a check over these files to look for accidental edits/modifictions